### PR TITLE
Fix - ignoring the first line in the log file

### PR DESCRIPTION
**Pull request by SivaMaplelabs**

### CODE REVIEW

File: platform/src/main/java/org/hillview/storage/GenericLogs.java [hunk 1]
```diff
@@ -84,7 +84,9 @@ public ITable load() {
                     this.error("File " + this.filename + " is empty!");
                 if (this.genLog.schema.getColumnCount() == 0) {
                     this.genLog.schema.append(new ColumnDescription("Host", ContentsKind.String));
+                    reader.mark(1);
                     String line = reader.readLine();
+                    reader.reset();
```
Comment (mbudiu-vmw) [inline platform/src/main/java/org/hillview/storage/GenericLogs.java]: Isn't it simpler just to reuse the current value of the line in the while loop below?

Comment (SivaMaplelabs) [reply platform/src/main/java/org/hillview/storage/GenericLogs.java]: @mbudiu-vmw Done.

**Comment by mbudiu-vmw** [comment]
Actually I will submit a PR which subsumes your fix soon for your review.

**Comment by SivaMaplelabs** [comment]
@mbudiu-vmw You want me to close this PR?

**Comment by mbudiu-vmw** [comment]
Let's wait until you review mine.

**Comment by mbudiu-vmw** [comment]
Hope to submit it a bit later today.

**Comment by mbudiu-vmw** [comment]
I will close this PR, and we'll replace it with the other one.

**Comment by SivaMaplelabs** [comment]
@mbudiu-vmw Sure

**Pull request closed without merging**